frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -319,12 +319,12 @@
             <td class="numeric">{{ execution(run) }}</td>
             <td class="numeric">
               <font-awesome-icon
-                v-if="statusOutput(run) === 1"
+                v-if="outputIconColorStatus(run) === 1"
```

Comment:
```suggestion
                v-if="outputIconColorStatus(run) === OutputStatus.Correct"
```

quizás necesitemos hacer

```typescript
OutputStatus = OutputStatus;
```

en L524 para poder usar `OutputStatus` acá.